src/WhatToCook.Application/Domain/Recipe.cs

```diff
@@ -56,7 +56,7 @@ public void SetImage(string imagePath)
         this.Image = imagePath;
     }
 
-    public void RemoveImage(string imagesDirectory)
+    public async void RemoveImage(string imagesDirectory)
```

Comment:
Avoid `async void` construct. You can read about it here:
http://www.pzielinski.com/?p=2928